src/impl/vamp/planning/simplify.hh

```diff
@@ -67,7 +67,7 @@ namespace vamp::planning
         const auto max_steps = (not settings.max_steps) ? path.size() : settings.max_steps;
         const auto max_empty_steps = (not settings.max_empty_steps) ? path.size() : settings.max_empty_steps;
 
-        rng::RNG rng;
+        rng::Random random;
```

Comment:
The mixed use of `rng` vs. the `random` object throughout here is confusing, and seems like it can cause bugs with seeding (i.e., the `random` object appears to always use the same default seed).